src/index.html

```diff
@@ -9,7 +9,56 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
+  <!-- <h1>HTML Form</h1>
+  <script type="text/javascript" src="./main.js"></script> -->
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+    <fieldset class="input-info"><legend>Personal information</legend>
+      <label class="row-elements" for="surname">Surname:
+      <input type="text" name="surname" id="surname" required autocomplete="off"></label>
```

Comment:
```suggestion
      <input type="text" name="surname" id="surname" required autocomplete="off">
    </label>
```
move parent tags to new line